src/services/createTables.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const { User } = require('./usersService');
+const { Expense } = require('./expensesService');
+
+User.sync({ force: true });
+Expense.sync({ force: true });
```

Comment:
It shouldn't be a separate service for this logic, just put it into the users/expenses services

Reply:
Why is this in the services? This file is run once when creating tables, and then it is better to delete it to avoid resetting tables. It is also called table creation.